src/misc/sw-template.js

```diff
@@ -278,6 +279,38 @@ function WebpackServiceWorker(params) {
     // * urlString -- url to match from the CACHE_NAME
     // * event.request -- original Request to perform fetch() if necessary
 
+    if (responseStrategy === "network-first") {
+      event.respondWith(
+        fetch(event.request)
+        .then((response) => {
+          if (DEBUG) {
+            console.log('[SW]:', `URL [${ urlString }] from network`);
+          }
+
+          // always fresh cache response for future usage
+          caches.open(CACHE_NAME).then((cache) => {
```

Comment:
You must change for response to be valid first, before adding it to the cache. `response.ok` check is fine.